pkg/proxy/winkernel/proxier.go

```diff
@@ -724,7 +725,7 @@ func NewProxier(
 		}
 		sourceVip = config.SourceVip
 		if len(sourceVip) == 0 {
-			return nil, fmt.Errorf("source-vip flag not set")
+			klog.InfoS("source-vip flag not set in the winkernel configuration, will check if an HNS endpoint exists with the well-known name %s", SOURCE_VIP_NAME)
```

Comment:
Can we move the logic to look it up by name up here?  The sourcevip is required and should be loaded during the initialization phase here instead of after it already started up so we can fail early.

Reply:
Yes, will be making this change.